dev/com.ibm.websphere.appserver.features/visibility/public/connectors-2.0/io.openliberty.connectors-2.0.feature

```diff
@@ -12,5 +12,5 @@ Subsystem-Category: JakartaEE9Application
  com.ibm.websphere.appserver.eeCompatible-9.0
 -bundles=com.ibm.ws.app.manager.rar, \
  com.ibm.ws.jca.1.7.jakarta
-kind=noship
-edition=full
+kind=beta
+edition=core
```

Comment:
JCA should be `edition=base` for as much as many features as we can. If there are features in `core` that require the JCA 2.0 API then we can leave the API feature in `core`, but aside from that we should move the features to `base`